source/Extractors/src/ExtractorFCCSW.cc

```diff
@@ -158,8 +158,8 @@ bool ExtractorFCCSW::analyze()
   xmlSubdetectorBeampipe->SetAttribute("name", "beampipe");
   xmlSubdetectorBeampipe->SetAttribute("type", "BeamTube");
   auto xmlBeamPipeDimensions = m_xmlDoc->NewElement("dimensions");
-  xmlBeamPipeDimensions->SetAttribute("rmin", printWithUnit(0., c_precision, "mm").c_str());
-  xmlBeamPipeDimensions->SetAttribute("rmax", printWithUnit(15., c_precision, "mm").c_str());
+  xmlBeamPipeDimensions->SetAttribute("rmin", printWithUnit(20., c_precision, "mm").c_str());
+  xmlBeamPipeDimensions->SetAttribute("rmax", printWithUnit(21., c_precision, "mm").c_str());
   xmlBeamPipeDimensions->SetAttribute("z", printWithUnit(500., c_precision, "mm").c_str());
   xmlBeamPipeDimensions->SetAttribute("material", "Air");
```

Comment:
You may know that, but just in case. The BP dimensions are defined here:
https://github.com/HEP-FCC/FCCSW/blob/master/Detector/DetFCChhBaseline1/compact/FCChh_DectDimensions.xml
z for the central part should be set to the same number as defined in the file: 800mm

One more comment, I haven't noticed that before. The BP wall is from Beryllium, not air :-) Other stuff should be fine ... Why do you by the way define the beam-pipe here? It's not problem, just would like to know...